docs/notes/2.29.x.md

```diff
@@ -20,7 +20,7 @@ Adds a label to `docker_environment` containers so pantsd can cleanup any old, d
 
 Fixed an issue where environment targets with empty sequence fields would override global configuration instead of inheriting from it. This affected multiple backends including Docker, Python, and NodeJS. For the Docker backend, empty `docker_env_vars` fields in `docker_environment` targets would prevent inheritance from global `[docker].env_vars` settings, causing Docker buildx to fail due to missing required environment variables like `HOME`. (See [#20605](https://github.com/pantsbuild/pants/issues/20605))
 
-
+Added [a new tool](https://github.com/pantsbuild/pants/tree/main/build-support/bin/external_tool_upgrade.py) to help with TemplatedExternalTool upgrades.
```

Comment:
Isn't this tool just for Pants developers and not end users? You need to clone the Pants repo to use it. If so I don't think we want to mention it in the notes (instead we can label the PR as `release-notes:not-required`. 

Unless... could this tool could be useful for plugin developers? If so we should mention here that this is just for that audience.

Reply:
Yeah, true, let's remove the notes for now.

PS I was thinking to extend the tool, so that it can be used to set whatever version you want in `pants.toml`, but it's not that easy, because you don't have enough information in `pants.toml` to fetch releases, so for now you have to run it on python backend and then copypaste